### PR TITLE
change socket communicating encoding to UTF-8

### PR DESCRIPTION
**Pull request by tantalate**
Should we use UTF-8 to support Unicode characters?

### CODE REVIEW

**Comment by neuhaus** [comment]
Have you tried this? Does it work? It seems like a good idea!

**Comment by tantalate** [comment]
I have tested it and it works on my server.

**Comment by neuhaus** [comment]
Awesome. I ran into a problem with posting some german umlauts into the chat as well. I guess this would fix the issue.

File: src/main/resources/mcpi/api/python/modded/mcpi/util.py [hunk 1]
```diff
@@ -15,4 +15,4 @@ def _misc_to_bytes(m):
 
     See `Connection.send` for more details.
     """
-    return str(m).encode("cp437")
+    return str(m).encode("UTF-8")
```
Comment (martinohanlon) [inline src/main/resources/mcpi/api/python/modded/mcpi/util.py]: Have you tested what would happen if you used the 'standard' mcpi python library? Does it still work?

**Comment by martinohanlon** [comment]
this will be in v1.12

**Pull request closed without merging**